README.md

```diff
@@ -82,7 +82,7 @@ js-wacz create --file "collection/*.warc"
 
 ### --output, -o
 
-Allows to specify where the resulting `.wacz` file should be created, and what its filename should be.
+Specify where the resulting `.wacz` file should be created, and what its filename should be.
```

Comment:
My brain wants to use "Specifies" and I am not sure why 🤔 . What do you think @bensteinberg ?

Reply:
I could see it either way. There could be an implicit "This option": "(This option) Specifies" -- or an implicit "This option allows you to" or "With this option you can": "(This option allows you to) Specify".

Reply:
I think I incline toward the latter in the interest of saving precious characters ;) but I'm happy to change these.

Reply:
Nope, that makes sense to me, let's ship it :D ! thanks again